v2/src/css/custom.css

```diff
@@ -519,3 +519,48 @@ html[data-theme="dark"] .markdown blockquote > *:last-child {
     bottom: 144px !important;
   }
 }
+
+/* Styles for collapsible elements */
+details {
+  position: relative;
+  transition: 0.3s, 0s border;
+  margin: 6px 0;
```

Comment:
Why is this margin here?

Reply:
to prevent two collapsible to stick to each other vertically.

So, I added margin, and compensate padding